pack/td.json

```diff
@@ -434,6 +528,40 @@
         "uniqueness": false
     },
     {
+        "advancement_cost": 5,
+        "agenda_points": 3,
+        "code": "13030",
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "illustrator": "Matt Zeilinger",
+        "keywords": "Initiative",
+        "pack_code": "td",
+        "position": 30,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "Place 2 agenda counters on Elective Upgrade when you score it.\n[click], <strong>Hosted agenda counter</strong>: Gain [click][click]. Use this ability only once per turn.",
```

Comment:
**[Minor Typo/nit pick]** '**_hosted_** agenda counter...' is typed lowercase in the card's text.